src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write your code here
+let sum = 0;
+let count = 0;
+
+document.querySelectorAll('.population').forEach((item) => {
+  sum += parseInt(item.innerHTML.split(',').join(''));
+  count++;
+});
+
+function numberWithCommas(x) {
+  return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+}
```

Comment:
use number.toLocalString() passing a correct argument or Intl to format a number